azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_mqtt.py

```diff
@@ -45,89 +69,47 @@ def _run_op(self, op):
                 ca_cert=self.ca_cert,
                 x509_cert=self.client_cert,
             )
-            self.transport.on_mqtt_connected = self.on_connected
-            self.transport.on_mqtt_disconnected = self.on_disconnected
-            self.transport.on_mqtt_message_received = self._on_message_received
+            self.transport.on_mqtt_connected = self._handle_mqtt_connected
+            self.transport.on_mqtt_connection_failure = self._handle_mqtt_connection_failure
+            self.transport.on_mqtt_disconnected = self._handle_mqtt_disconnected
```

Comment:
Was there a particular reason we moved away from the `on_x` convention to a `handle_x` convention?

Reply:
Because we went through this whole thing a few months ago where went back and forth about what to call things and my memory wasn't in sync with the wiki.  I'll sync with the wiki.

---
In reply to: [309365971](https://github.com/Azure/azure-iot-sdk-python-preview/pull/136#discussion_r309365971) [](ancestors = 309365971)